pkgs/development/tools/analysis/hopper/default.nix

```diff
@@ -1,42 +1,57 @@
-{ stdenv, fetchurl, pkgs, lib }:
+{
+  stdenv, fetchurl, lib,
```

Comment:
This is little unwieldy, could you try to clean it up a bit (e.g. using nixpkgs-fmt).

Reply:
not a huge fan what the `nixpkgs-fmt` output looks like, but did it anyways.